_includes/_trip-grid.html

```diff
@@ -69,7 +69,7 @@ <h3 class="font-family-condensed-extra text-uppercase font-size-h4 push-half-top
             </div>
           </div>
           <div class="south-africa card push-ends col-xs-12 col-sm-4 col-md-3"><a href="https://www.crossroads.net/reachout/go/south-africa/june-trip">
-              <img data-src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-trips-bg-SA-trip2b.jpg" class="card-img-top img-responsive"
+              <img data-src="//images.ctfassets.net/y3a9myzsdjan/6laR9QJJzmM8BOuq9OrYYv/0b1ad491765dee969cac0d47d51b05fe/26.jpg" class="card-img-top img-responsive"
```

Comment:
You may want to use the imgix source: `http://crds-media.imgix.net/6laR9QJJzmM8BOuq9OrYYv/0b1ad491765dee969cac0d47d51b05fe/26.jpg?w=800&format=auto`